### ISSUE TITLE
Mathtype adds extra br elements to the output markup

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#2013]
## Is this a bug report or feature request?

🐞 Bug report

## 💻 Version of CKEditor

CKEditor v5 @ 12.4.0

## 📋 Steps to reproduce

1. Use an editor with mathtype and table plugins.
2. Add a 2x2 table.
3. Call `editor.getData()`.

## ✅ Expected result

Table markup without `br`s.

```html
<table>
	<tbody>
		<tr>
			<td>&nbsp;</td>
			<td>&nbsp;</td>
		</tr>
		<tr>
			<td>&nbsp;</td>
			<td>&nbsp;</td>
		</tr>
	</tbody>
</table>
```

## ❎ Actual result

Returned table has some extra `br`s in it.

```html
<table>
	<tbody>
		<tr>
			<td><br data-cke-filler="true"></td>
			<td><br data-cke-filler="true"></td>
		</tr>
		<tr>
			<td><br data-cke-filler="true"></td>
			<td><br data-cke-filler="true"></td>
		</tr>
	</tbody>
</table>
```

**Comment by Reinmar** [comment]
DUP reported in https://github.com/ckeditor/ckeditor5/issues/5589.

**Comment by FilipTokarski** [comment]
I checked it with the updated MathType version and it seems to be working fine now.

**Comment by FilipTokarski** [comment]
This issue is fixed in the new MathType version 7.19.0
